benchmarks/diffusion_2D/main_arkode.cpp

```diff
@@ -545,39 +640,56 @@ void UserOptions::print()
   cout << " --------------------------------- " << endl;
   cout << " rtol        = " << rtol << endl;
   cout << " atol        = " << atol << endl;
-  cout << " hfixed      = " << hfixed << endl;
-  cout << " order       = " << order << endl;
   cout << " controller  = " << controller << endl;
-  cout << " max steps   = " << maxsteps << endl;
-  cout << " linear RHS  = " << linear << endl;
+  cout << " hfixed      = " << hfixed << endl;
   cout << " --------------------------------- " << endl;
-
   cout << endl;
-  if (ls == "sludist")
+  if (implicit)
   {
-    cout << " Linear solver options:" << endl;
+    cout << " ARKStep options:" << endl;
     cout << " --------------------------------- " << endl;
+    cout << " order       = " << order << endl;
+    cout << " max steps   = " << maxsteps << endl;
+    cout << " linear RHS  = " << linear << endl;
+    cout << " --------------------------------- " << endl;
+    cout << endl;
+    if (ls == "sludist")
+    {
+      cout << " Linear solver options:" << endl;
+      cout << " --------------------------------- " << endl;
 #if defined(HAVE_HIP)
-    cout << " LS       = SuperLU_DIST (HIP enabled)" << endl;
+      cout << " LS       = SuperLU_DIST (HIP enabled)" << endl;
 #elif defined(HAVE_CUDA)
-    cout << " LS       = SuperLU_DIST (CUDA enabled)" << endl;
+      cout << " LS       = SuperLU_DIST (CUDA enabled)" << endl;
 #else
-    cout << " LS       = SuperLU_DIST" << endl;
+      cout << " LS       = SuperLU_DIST" << endl;
 #endif
-    cout << " LS info  = " << lsinfo << endl;
-    cout << " msbp     = " << msbp << endl;
-    cout << " --------------------------------- " << endl;
+      cout << " LS info  = " << lsinfo << endl;
+      cout << " msbp     = " << msbp << endl;
+      cout << " --------------------------------- " << endl;
+    }
+    else
+    {
+      cout << " Linear solver options:" << endl;
+      cout << " --------------------------------- " << endl;
+      cout << " LS       = " << ls << endl;
+      cout << " precond  = " << preconditioning << endl;
+      cout << " LS info  = " << lsinfo << endl;
+      cout << " LS iters = " << liniters << endl;
+      cout << " msbp     = " << msbp << endl;
+      cout << " epslin   = " << epslin << endl;
+      cout << " --------------------------------- " << endl;
+    }
   }
   else
   {
-    cout << " Linear solver options:" << endl;
+    cout << " LSRKStep options:" << endl;
     cout << " --------------------------------- " << endl;
-    cout << " LS       = " << ls << endl;
-    cout << " precond  = " << preconditioning << endl;
-    cout << " LS info  = " << lsinfo << endl;
-    cout << " LS iters = " << liniters << endl;
-    cout << " msbp     = " << msbp << endl;
-    cout << " epslin   = " << epslin << endl;
+    switch (lsrkmethod)
+    {
+    case (ARKODE_LSRK_RKC_2): cout << " method = RKC_2 " << endl; break;
+    case (ARKODE_LSRK_RKL_2): cout << " method = RKL_2 " << endl; break;
```

Comment:
This covers all of the methods that this executable can be run with (I added some checks to the beginning of `main` to error out if run with something else).

I'll defer to Mustafa on whether he wants to create a something similar to `ARKodeButcherTable_ERKIDToName` for LSRKStep at this time.  While that might be nice for our own testing purposes, I don't see it as particularly necessary for users, and I think it could be added at a future date.